src/db.js

```
@@ -15,18 +14,13 @@ const {
   POSTGRES_DB,
 } = process.env;
 
-/*
-  All credentials setted to default values (exsept password - it is exapmle)
-  replace if needed with your own
-*/
-
 const sequelize = new Sequelize({
   database: POSTGRES_DB || 'postgres',
   username: POSTGRES_USER || 'postgres',
   host: POSTGRES_HOST || 'localhost',
   dialect: 'postgres',
   port: POSTGRES_PORT || 5432,
-  password: POSTGRES_PASSWORD || '123',
+  password: POSTGRES_PASSWORD || '1815',
```

Avoid hard-coding a default DB password. Using `'1815'` here can mask misconfiguration and is a security risk. Prefer leaving `password` undefined when the env var is missing so startup fails loudly, or clearly document why a default is necessary for local testing. Example alternatives: `password: POSTGRES_PASSWORD` or `password: process.env.POSTGRES_PASSWORD || undefined`.